src/plugin.ts

```diff
@@ -14,4 +21,23 @@ export const plugin = createPlugin({
       factory: ({ discoveryApi }) => new TravisCIApiClient({ discoveryApi }),
     }),
   ],
+  routes: {
+    entityContent: entityContentRouteRef,
+  },
 });
+
+export const EntityArgoCDContent = travisciPlugin.provide(
```

Comment:
There two names should be travis I guess

Reply:
Fixed